packages/grid/rathole/start.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+MODE=${MODE:-server}
+RUST_LOG=${LOG_LEVEL:-trace}
+
+# Copy configuration files
+copy_config() {
+  cp -L -r -f /conf/* conf/
```

Comment:
copies the mounted data from rathole configmap to the respective working directory.